externs/olcsx.js

```diff
@@ -5,6 +5,45 @@
  */
 var olcsx;
 
+
+/**
+ * @typedef {{
+ *   map: (!ol.Map),
+ *   target: (Element|string|undefined),
+ *   createSynchronizers: (undefined|function(!ol.Map, !Cesium.Scene): Array.<olcs.AbstractSynchronizer>)
```

Comment:
In the case of functions (which we have here) the same is done in ol3. Otherwise you need more braces.